### PR TITLE
[Merged by Bors] - Bump operator-rs to 0.27.1

### PR DESCRIPTION
**Pull request by sbernauer**
# Description

Fixes https://github.com/stackabletech/commons-operator/issues/113

## Review Checklist

- [x] Code contains useful comments
- [x] CRD change approved (or not applicable)
- [x] (Integration-)Test cases added (or not applicable)
- [x] Documentation added (or not applicable)
- [x] Changelog updated (or not applicable)
- [x] Cargo.toml only contains references to git tags (not specific commits or branches)
- [x] Helm chart can be installed and deployed operator works (or not applicable)

Once the review is done, comment `bors r+` (or `bors merge`) to merge. [Further information](https://bors.tech/documentation/getting-started/#reviewing-pull-requests)

### CODE REVIEW

**Comment by maltesander** [comment]
Can you bump to 0.27.1? 0.27.0 had a label bug https://github.com/stackabletech/operator-rs/pull/507

**Review by maltesander — APPROVED** [review]
LGTM!

File: rust/operator-binary/src/restart_controller/statefulset.rs [hunk 1]
```diff
@@ -85,30 +84,29 @@ pub async fn start(client: &Client) {
...
             stream::select(
                 trigger_all(
-                    try_flatten_touched(
-                        reflector(cm_store, watcher(cms, ListParams::default())).inspect(|_| {
-                            cms_inited.store(true, std::sync::atomic::Ordering::SeqCst)
-                        }),
-                    ),
+                    reflector(cm_store, watcher(cms, ListParams::default()))
+                        .inspect(|_| cms_inited.store(true, std::sync::atomic::Ordering::SeqCst))
+                        .applied_objects(),
```
Comment (teozkr) [inline rust/operator-binary/src/restart_controller/statefulset.rs:89]: This should be `touched_objects()`, not `applied_objects()`.

File: rust/operator-binary/src/restart_controller/statefulset.rs [hunk 2]
```diff
@@ -85,30 +84,29 @@ pub async fn start(client: &Client) {
...
-                    try_flatten_touched(
-                        reflector(secret_store, watcher(secrets, ListParams::default())).inspect(
-                            |_| secrets_inited.store(true, std::sync::atomic::Ordering::SeqCst),
-                        ),
-                    ),
+                    reflector(secret_store, watcher(secrets, ListParams::default()))
+                        .inspect(|_| {
+                            secrets_inited.store(true, std::sync::atomic::Ordering::SeqCst)
+                        })
+                        .applied_objects(),
```
Comment (teozkr) [inline rust/operator-binary/src/restart_controller/statefulset.rs:97]: And here.

File: rust/operator-binary/src/restart_controller/statefulset.rs (hunk 1 above)
Comment (sbernauer) [reply rust/operator-binary/src/restart_controller/statefulset.rs:89]: thx! Fixed in 9e14f9075c94887dd8151e219e3c0443010961ef

File: rust/operator-binary/src/restart_controller/statefulset.rs (hunk 2 above)
Comment (sbernauer) [reply rust/operator-binary/src/restart_controller/statefulset.rs:97]: thx! Fixed in 9e14f9075c94887dd8151e219e3c0443010961ef

**Pull request closed without merging**